lib/rules/web/ocm_console/cluster_detail.xyaml

```diff
@@ -224,47 +224,47 @@ check_ocp_tab_in_detail_page:
 check_ocp_usage_in_detail_page:
     elements:
         - selector:
-            xpath: //h2[@class='pf-c-title card-title' and text()='Resource usage']
+            xpath: //h2[text()='Resource usage']
         - selector:
             xpath: //p[text()='The cluster currently does not have any metrics data. Try again later.']
 check_ocp_details_in_detail_page:
     elements:
         - selector:
-            xpath: //h2[@class='pf-c-title card-title' and text()='Details']
+            xpath: //h2[text()='Details']
         - selector:
-            xpath: //dl[@class='cluster-details-item left']/dt[text()='Cluster ID']
+            xpath: //dl[@class='cluster-details-item']/dt[text()='Cluster ID']
```

Comment:
Better not to use the class to locate. It's better to use //dl[1]/dt[]

Reply:
Updated